packages/eyes/src/eyes-test/eyes.ts

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const glob = require('glob');
+const chalk = require('chalk');
+const dotenv = require('dotenv');
+const {JSDOM} = require('jsdom');
+const {makeVisualGridClient, initConfig} = require('@applitools/visual-grid-client');
+const domNodesToCdt = require('@applitools/visual-grid-client/src/browser-util/domNodesToCdt');
+const {TestFailedError, TestResults} = require('@applitools/eyes.sdk.core');
+
+dotenv.config();
+
+function getGridClientConfig() {
+  const projectName = 'kieran-grid-test';
+  const branchName = projectName + '/master';
+
+  return {
+    appName: projectName,
+    branchName,
+    batchName: projectName,
+    browser: {
+      name: 'chrome',
+      width: 400,
+      height: 300
+    }
+  };
+}
+
+function getStaticResources(cssFilenames: string[], resourceDir: string) {
+  const resources: any = {};
+  for (const cssFilename of cssFilenames) {
+    const url = '/' + cssFilename;
+    resources[url] = {
+      url,
+      type: 'text/css',
+      value: fs.readFileSync(path.join(resourceDir, cssFilename))
+    };
+  }
+  return resources;
+}
+
+function getStatusIcon(result: any) {
```

Comment:
`result: string`, and there's a lot of Anies below for which we can also specify the type.